src/manage_nsfs/manage_nsfs_cli_errors.js

```diff
@@ -141,6 +141,30 @@ ManageCLIError.WhiteListIPUpdateFailed = Object.freeze({
     http_code: 500,
 });
 
+ManageCLIError.InvalidMasterKey = Object.freeze({
+    code: 'InvalidMasterKey',
+    message: 'Master key manager had issues loading master key, can not decrypt/encrypt secrets.',
+    http_code: 500,
+});
+
+ManageCLIError.InvalidMasterKeyStoreType = Object.freeze({
+    code: 'InvalidMasterKeyStoreType',
```

Comment:
the question if more specific codes are needed here because any client is interested to differentiate between these cases? if these are just for information then using the same code with a different message is better because it will allow clients to identify this error with a single error code...

Reply:
I wanted to make sure the client won't receive InternalError code and have a clear understanding of the failure.
A single error code and different error messages should be enough, thanks!